lxd/instance/instance_utils.go

```diff
@@ -18,7 +18,7 @@ import (
 	liblxc "github.com/lxc/go-lxc"
 	"github.com/pborman/uuid"
 
-	"github.com/lxc/lxd/client"
+	lxd "github.com/lxc/lxd/client"
```

Comment:
This needs to be removed. The import organizer tool does this, but we don't want it.

Reply:
ah yes.. this is sth I thought I removed before

Reply:
Yeah, this happened on the other PR you put up yesterday. Probably need to disabling something in vscode.

Reply:
> Yeah, this happened on the other PR you put up yesterday. Probably need to disabling something in vscode.

I haven't found a way of doing this without disabling the autoimport feature which is quite handy.

Reply:
Yep I disabled that I think.